<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Portfolio of Manuel Cabrera Justo</title>
    <link rel="stylesheet" href="styles.css">
</head>
<body>
    <header>
	<img src="profile.jpg" alt="Manuel Cabrera Justo" class="profile-pic">
	<h1>Manuel Cabrera Justo</h1>
        <h2>Junior GenAI Developer</h2>
        <p>Email: <a href="mailto:manucabreraj7@gmail.com">manucabreraj7@gmail.com</a></p>
        <p>Phone: +34 666633364</p>
        <p>GitHub: <a href="https://github.com/Mcabreraj7" target="_blank">Mcabreraj7</a></p>
    </header>
    
    <section>
        <h2>Projects</h2>
        <article>
            <h3>K-means clustering</h3>
            <p>This Python script utilizes K-means clustering to analyze bacterial gene expression data, allowing users to dynamically specify the number of strains for comparison and visualize clustering results using PCA.</p>
            <p><a href="https://github.com/Mcabreraj7/k-means-clustering" target="_blank">View on GitHub</a></p>
        </article>
	    <h3>News summarizer</h3>
            <p>This Python script scrapes textual content from a web page and generates a concise summary using a transformer-based model BART</p>
            <p><a href="https://github.com/Mcabreraj7/news-summarizer" target="_blank">View on GitHub</a></p>
        </article>
	    <h3>GC-content predictor</h3>
            <p>A Python script for predicting the Guanine/Cytosine (GC) content of DNA sequences using linear regression, extracting random fragments from a genome and analyzing their nucleotide composition.</p>
            <p><a href="https://github.com/Mcabreraj7/GC-content-predictor" target="_blank">View on GitHub</a></p>
        </article>
	    <h3>Antibiotic-resistance indicator</h3>
            <p>Using a Random Forest classifier, this script predicts antibiotic resistance by analyzing feature data and evaluating the model’s performance through a classification report.</p>
            <p><a href="https://github.com/Mcabreraj7/Antibiotic-resistance-indicator" target="_blank">View on GitHub</a></p>
        </article>
    </section>
    <section>
        <h2>Professional Experience</h2>
        <article>
            <h3>Junior GenAI Developer - Pampling</h3>
            <p><em>Seville, Spain (Nov 2021 - Mar 2024)</em></p>
            <ul>
                <li>T-shirt design generator leveraging Stable diffusion in AWS</li>
                <li>Managed and analyzed customer databases using SQL and Python, extracting insights to optimize marketing strategies.</li>
            </ul>
        </article>
        <article>
            <h3>Laboratory Technician - Microal S.L.</h3>
            <p><em>Bollullos de la Mitación, Seville, Spain (Feb 2021 - May 2021)</em></p>
            <ul>
                <li>Extracurricular internship in Physicoquemical Analysis Department.</li>
                <li>Processed and analyzed biological datasets from food and water, using chromatographic techniques (GC/ECD, GC/FID, HPLC) for quantitative analysis of fatty acids, sugars, and hydrocarbons.</li>
                <li>Performed statistical analysis (ANOVA, regression) and created data visualizations to identify patterns and trends in extracted data.</li>
            </ul>
        </article>
        <article>
            <h3>Laboratory Associate - Weisse lab, University of Edinburgh</h3>
            <p><em>Edinburgh, United Kingdom (Jan 2020 - May 2020)</em></p>
            <ul>
                <li>Conducted whole-genome sequencing and phylogenomic analysis to investigate bacterial antimicrobial resistance and hospital transmission dynamics.</li>
                <li>Applied K-means clustering to classify bacterial isolates based on genomic and antimicrobial resistance profiles and used Decision Trees to identify key factors influencing resistance and transmission.</li>
            </ul>
        </article>
	<article>
            <h3>Laboratory Analyst at Andalusian Center for Developmental Biology (CABD)</h3>
            <p><em>Seville, Spain (Sep 2018 - Aug 2019)</em></p>
            <ul>
                <li>Engineered a mutant bacterial strain using cloning techniques combined with computational analysis.</li>
            <li>Carried out gene expression studies (Q-PCR) to compare both wild-type and mutant strains.</li>
            <li>Python data pipelines (using Pandas, NumPy, Matplotlib, Seaborn, and Jupyter Notebooks) enabling efficient dataset handling, reproducibility, and clear visualizations for data-driven decision-making.</li>
            </ul>
        </article>
	<article>
            <h3>Research Intern at Departament of Physiology (University of Seville)</h3>
            <p><em>Seville, Spain (Sep 2015 - Jun 2017)</em></p>
            <ul>
               <li>Academic research for the Bachelor's Thesis titled "The Role of Dopamine in Autism" using biomedical databases like PubMed.</li>
               <li>Support work in my supervisor's research conducting molecular biology techniques such as DNA isolation, PCR, and ELISA assay.</li>
            </ul>
        </article>
    </section>
    
    <section>
        <h2>Academic Background</h2>
        <ul>
            <li>MSc. Environmental, Industrial and Food Biotechnology - University of Pablo de Olavide (2017-2019)</li>
            <li>Bachelor’s Degree of Science in Biology - University of Seville (2010-2015)</li>
        </ul>
    </section>
    
    <section>
        <h2>Courses</h2>
        <ul>
            <li>ChatGPT Prompt Engineering for Developers - DL.ai (2023)</li>
            <li>Generative AI with LLM’s - DL.ai (2023)</li>
            <li>Introduction to Version Control with Git and GitHub - University of the Andes (2023)</li>
            <li>Machine Learning in Production - DL.ai (2022)</li>
            <li>Data Science and Data Analyst - INESEM Business School (2021)</li>
        </ul>
    </section>
    
    <section>
        <h2>Languages</h2>
        <ul>
            <li>English – Professional (B2)</li>
            <li>Spanish - Native</li>
        </ul>
    </section>
    
    <section>
        <h2>Technical Skills</h2>
        <ul>
            <li>Python: Numpy, Pandas, Matplotlib, HuggingFace transformers, AWS Bedrocks</li>
            <li>MongoDB, SQL</li>
	    <li>Git</li>	
            <li>Linux</li>
	    <li>Weights & Biases</li>
	    <li>NLP/GenAI: Embeddings, RAG, Vector databases</li>
        </ul>
    </section>
    
    <footer>
        <p>&copy; 2025 Manuel Cabrera Justo</p>
    </footer>
</body>
</html>
